projects/storefrontlib/cms-components/navigation/breadcrumb/breadcrumb.component.ts

```diff
@@ -36,6 +53,19 @@ export class BreadcrumbComponent extends PageTitleComponent implements OnInit {
   ngOnInit(): void {
     super.ngOnInit();
     this.setCrumbs();
+    if (this.featureConfigService.isEnabled('a11yRepeatedPageTitleFix')) {
+      this.subscription.add(
+        this.router.events
+          .pipe(filter((e) => e instanceof NavigationEnd))
+          .subscribe(() => {
+            if (document.activeElement === document.body) {
+              this.ariaLiveEnabled = false;
```

Comment:
Maybe we could make this flag an observable instead and use the async pipe for it in the template for `[ariaLive]`. Should save us all the subscription logic.

Reply:
You are right I have simplified it a bunch.